spk/python310/src/requirements.txt

```diff
@@ -0,0 +1,52 @@
+# Basic wheels
+setuptools==58.3.0
+pip==21.3.1
+wheel==0.37.0
+
+# cryptography dependencies
+idna==3.3
+ipaddress==1.0.23
```

Comment:
This one is now part of the base python library since 3.3 and can be removed.

Reply:
this one meaning `ipaddress` ?

Reply:
after looking for a bit, indeed `ipaddress` looks like already included.  removing.